.github/in-cluster-test-script.sh

```diff
@@ -11,4 +11,4 @@ cilium status --wait
 
 cilium hubble port-forward&
 
-cilium connectivity test
+cilium connectivity test --all-flows
```

Comment:
Why don't we make `--all-flows` behaviour the default? Or a flag `--ci-run` that enables some specific verbosities we want on CI without piling on arguments in the workflows here, that will get messy.